Sources/OpenWeatherKit/Public/Forecast/HourWeather.swift

```diff
@@ -54,6 +54,11 @@ public struct HourWeather: Sendable {
     /// The pressure trend, or barometric tendency, is the kind and amount of atmospheric pressure
     /// change over time.
     public var pressureTrend: PressureTrend
+    
+    /// The rate at which snow crystals are falling, in millimeters per hour.
+    public var snowfallIntensity: Measurement<UnitSpeed>?
+    
+    public var snowfallAmount: Measurement<UnitLength>?
```

Comment:
The native `WeatherKit` [docs](https://developer.apple.com/documentation/weatherkit/hourweather/snowfallamount) state 

```
Snowfall amount refers to the depth of snow rather than the liquid equivalent. The system computes this value by applying a temperature-based conversion to crystal size.
```

I don't see where is this value is being set. I think this value will need to be computed or removed.

Reply:
I am pretty sure I got an snowfallAmount back from the Apple servers. That's why I added it. I try to reproduce this and come back.

Reply:
Interesting that intensity is optional in the [api docs](https://developer.apple.com/documentation/weatherkitrestapi/hourweatherconditions) and amount is absent while the opposite is true for the [native framework](https://developer.apple.com/documentation/weatherkit/hourweather/snowfallamount).

Maybe the safest thing to do is treat `snowfallIntensity` and `snowfallAmount` in the internal api payload as optionals, and if they are null, set there measurement values to `0`.

Reply:
I am pretty sure that, when I looked at it a couple of weeks ago, these two values were optional. Today I get these two values even for regions where never has been snow. But maybe I have been too fast, making the two values mandatory. I can revert, the last commit.